app/directives/proposals/proposals.directive.js

```diff
@@ -93,6 +112,9 @@ angular.module('dashboard')
                     if (proposal.isNew) {
                         delete proposal.isNew;
                     }
+                    if (proposal.isEditing) {
```

Comment:
What if property value is `false`.
Could `isNew` and `isEditing` be deleted always without an `if`?